cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -499,3 +499,454 @@ def test_create_snapshot_via_ae(appliance, request, domain, small_test_vm):
 
     # Clean up if it appeared
     snapshot.delete()
+
+
+@pytest.mark.manual
+@test_requirements.snapshot
+@pytest.mark.tier(1)
+def test_rhos_notification_for_snapshot_failures():
+    """
+    Test if cfme can report failure when deleting or creating
+    snapshot on RHOS.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: Infra
+        caseimportance: medium
+        initialEstimate: 1/4h
+        title: RHOS: test notification for snapshot create/delete failure
+        setup:
+            1. Add RHOS provider
+            2. Create a test VM on RHOS provider
+        testSteps:
+            1. Try to introduce a failure while creating snapshot
+                * Delete the VM in the process of creating a snapshot
+            2. Try to introduce a failure while deleting snapshot
+                * Disconnect machine from network and then try to delete snapshot
+        expectedResults:
+            1. Notification for failure when creating the snapshot appears
+            2. Notification for failure when deleting the snapshot appears
+    Bugzilla:
+        1581793
+
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.snapshot
+@pytest.mark.tier(1)
+def test_notification_for_snapshot_actions_on_openstack():
```

Comment:
Again - not quite the right module for these tests, since its named for (and should collect against) openstack.